corehq/pillows/mappings/domain_mapping.py

```diff
@@ -211,6 +211,7 @@
                 'project_type': {'analyzer': 'comma', 'type': 'string'},
                 'published': {'type': 'boolean'},
                 'publisher': {'type': 'string'},
+                'allow_domain_requests': {'type': 'boolean'},
```

Comment:
updated index. i'm not using this in ES, but would rather the mapping be complete than be minimal. (guessing from its current size that keeping it minimal isn't a priority)